packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -697,6 +796,16 @@ class Viewport implements IViewport {
     this.initialCamera = camera;
   }
 
+  /**
+   * Sets the provided camera as the displayArea camera.
+   * This allows computing differences applied later as compared to the initial
+   * position, for things like zoom and pan.
+   * @param camera - to store as the initial value.
+   */
+  protected setInitialDisplayAreaCamera(camera: ICamera): void {
+    this.initialDisplayAreaCamera = camera;
+  }
+
```

Comment:
We already have the `initialCamera` that I hate very much, why we are adding this new initialDisplayCamera too?

Reply:
The initialCamera is the position the user will reset to
The initialDisplayCamera is the position prior to applying the initialDisplay (this allows intialDisplay settings to be absolute, otherwise subsequent applications will cause another zoom and another pan on top of what has already applied)